tests/cypress/tests/standalone/navigation_test.ts

```diff
@@ -28,90 +28,6 @@ Object.values(AppMode).forEach(mode => {
                 DashTable.getCell(3, 1).click();
             });
```

Comment:
navigation_test now covers the cases that are applicable to both readonly and readwrite flavors of the table